lib/tasks/webpacker.rake

```diff
@@ -3,16 +3,22 @@ WEBPACKER_APP_TEMPLATE_PATH = File.expand_path('../install/template.rb', __dir__
 namespace :webpacker do
   desc "Compile javascript packs using webpack for production with digests"
   task :compile => :environment do
-    webpack_digests_json = JSON.parse(`WEBPACK_ENV=production ./bin/webpack --json`)['assetsByChunkName'].to_json
+    dist_path = Rails.application.config.x.webpacker[:packs_dist_path]
+    webpack_digests = JSON.parse(`WEBPACK_DIST_PATH=#{dist_path} WEBPACK_ENV=production ./bin/webpack --json`)['assetsByChunkName']
+
+    js_webpack_digests_json = webpack_digests.each_with_object({}) do |(chunk, file), h|
+      h[chunk] = file.is_a?(Array) ? file.first : file
+    end.to_json
```

Comment:
This fix for case when you have to export source-map with your JS. In this case each chunk will be array [source.js, source.js.map]. So we have to cleanup it here.